config/app_dynamics_agent.yml

```diff
@@ -17,6 +17,6 @@
 ---
 version: 4.1.+
 repository_root: ! '{default.repository.root}/app-dynamics'
-default_application_name:
-default_node_name: ! '$(expr "$VCAP_APPLICATION" : ''.*instance_index[": ]*\([[:digit:]]*\).*'')'
-default_tier_name:
+default_application_name: 
+default_node_name: ! '$(expr "$VCAP_APPLICATION" : ''.*application_name[": ]*\([-_a-zA-Z0-9]*\).*''):$(expr "$VCAP_APPLICATION" : ''.*instance_index[": ]*\([[:digit:]]*\).*'')'
+default_tier_name: ! 
```

Comment:
I'm pretty sure that this should be an empty line.  What we want is the YAML equivalent of `null` there, not the YAML equivalent of an empty string.